tests/feature/test_tags.py

```diff
@@ -1,25 +1,23 @@
 """Test tags."""
-import pytest
-
-from pytest_bdd import scenario, feature
+import textwrap
 
+import pytest
 
-def test_tags(request):
-    """Test tags for the scenario and the feature."""
-    @scenario(
-        'tags.feature',
-        'Tags'
-    )
-    def test():
-        pass
-
-    assert test.__scenario__.tags == set(['scenario_tag_1', 'scenario_tag_2'])
-    assert test.__scenario__.feature.tags == set(['feature_tag_1', 'feature_tag_2'])
-    test(request)
```

Comment:
One last question... are you sure this test is not needed? ;-)

Reply:
I don't it's purpose. We already have other tests that assert that pytest "sees" the tags, by asking pytest to run only certain items.

We don't expose any API for test.__scenario__, so what are we testing it for?
Moreover, this test was calling `test(request)`, what is even the point of that?